src/fragments/lib/push-notifications/common/record-notifications.mdx

```diff
@@ -0,0 +1,13 @@
+The Amplify Push Notifications plugin provides a simple API to record the receipt and opening of notifications within your app.
```

Comment:
Maybe we should explain a little more _why_ you want to do. it, or _what_ does it achieve (i.e. engagement metrics from Pinpoint)

Reply:
yes, done.